requirements.txt

```diff
@@ -2,4 +2,4 @@ pyyaml
 msgpack
 joblib
 rich
-
+tqdm
```

Comment:
This won't cause breaks downstream because it is optional - correct?

Reply:
This was temporary and has been removed. In the future I would like to migrate away from the tqdm dependency since the custom progress bar and spinner provide good functionality without needing another library.